requirements.txt

```
@@ -8,4 +8,6 @@ py-cord~=1.7
 aiohttp~=3.7.4  # Waiting for py-cord to upgrade
 setuptools~=61.1
 
-win10toast~=0.9; sys_platform == "win32"
\ No newline at end of file
+win10toast~=0.9; sys_platform == "win32"
+
+feedparser~=6.0.8
```

```suggestion
feedparser~=6.0.8

```
Just adding a newline here in requirements.txt to avoid it thinking something changed with the prev line when nothing changed.

---

Very sensible - adopted